code/networking/src/socket_programming/README.md

```diff
@@ -0,0 +1,46 @@
+# Socket Programming using C Language in Linux operating system.
+
+## The client server model
+
+Most interprocess communication uses the client server model. These terms refer to the two processes which will be communicating with each other. One of the two processes, the client, connects to the other process, the server, typically to make a request for information.  
```

Comment:
Mention what is Interprocess Communication. And then mention that Socket Programming is a generalized type of IPC mechanism.